Documentation/ApiOverview/FeatureToggles/Index.rst

```diff
@@ -95,3 +102,23 @@ Internally, the changes are written to :file:`LocalConfiguration.php`::
       ],
    ]
 
+Feature Toogles in TypoScript
+===============================
+
+To check wether a feature is enabled in TypoScript was introduced in v9.5 in :issue:`86881`
+
+Support for feature toggle check in the symfony expression language DefaultFunctionProvider is provided.
+With the new function :typoscript:`feature()` the feature toggle can be checked.
+
+.. code-block:: typoscript
+
+   [feature("TypoScript.strictSyntax")]
+   # This condition matches if the feature toggle "TypoScript.strictSyntax" is true
+   [END]
+
+   [feature("TypoScript.strictSyntax") === false]
+   # This condition matches if the feature toggle "TypoScript.strictSyntax" is false
+   [END]
+
+
+.. index:: Backend, Frontend, TypoScript, ext:core
```

Comment:
We currently don't use index in "TYPO3 Explained". While this is probably a good idea, let's make a general decision if and how before we start doing that.